clients/admin-ui/src/features/common/ConfirmationModal.tsx

```diff
@@ -44,7 +44,7 @@ const ConfirmationModal = ({
     isOpen={isOpen}
     onClose={onClose}
     size="lg"
-    returnFocusOnClose={returnFocusOnClose || true}
+    returnFocusOnClose={returnFocusOnClose ?? true}
```

Comment:
@TheAndrewJackson - This was always resulting in `true` which made the buttons pop up after the modal closed.